locales/de-DE.json

```diff
@@ -479,10 +479,10 @@
         "unsupported": "Ihr Browser unterstützt keine Push-Benachrichtigungen.",
         "warning": {
           "enable_close": "Schließen",
-          "enable_description": "Um Benachrichtigungen zu erhalten, wenn Elk nicht geöffnet ist, aktiviere Push-Benachrichtigungen. \nDu kannst genau steuern, welche Arten von Interaktionen Push-Benachrichtigungen generieren, indem du die Schaltfläche \"@:settings.notifications.show_btn{'\"'} oben aktivierest, sobald sie aktiviert ist.",
-          "enable_description_desktop": "Um Benachrichtigungen zu erhalten, wenn Elk nicht geöffnet ist, aktiviere Push-Benachrichtigungen. \nDu kannst unter „Einstellungen > Benachrichtigungen > Einstellungen für Push-Benachrichtigungen“ genau steuern, welche Arten von Interaktionen Push-Benachrichtigungen generieren, sobald diese aktiviert sind.",
+          "enable_description": "Um Benachrichtigungen zu erhalten, wenn Mozilla.social nicht geöffnet ist, aktiviere Push-Benachrichtigungen. \nDu kannst genau steuern, welche Arten von Interaktionen Push-Benachrichtigungen generieren, indem du die Schaltfläche \"@:settings.notifications.show_btn{'\"'} oben aktivierest, sobald sie aktiviert ist.",
```

Comment:
It was already wrong in the previous version, but since you're already on it: "aktivierest" should be "aktivierst". But I am not sure if that's the correct word at all, you usually don't "enable" a button, and it's also not what the English original says.

Suggestion: "Um Benachrichtigungen zu erhalten, wenn Mozilla.social nicht geöffnet ist, aktiviere Push-Benachrichtigungen. \nÜber die Schaltfläche \"@:settings.notifications.show_btn{'\"'} oben kannst genau steuern, welche Arten von Interaktionen Push-Benachrichtigungen generieren, sobald diese aktiviert ist."

Reply:
Hi @cadeyrn, thanks for the suggestion! I've put your changes onto a new branch and will ask our product if these fixes can come in. PR is here - https://github.com/MozillaSocial/elk/pull/87